libs/sdk-backend-base/api/sdk-backend-base.api.md

```diff
@@ -56,9 +56,9 @@ import { IInsight } from '@gooddata/sdk-model';
 import { IInsightDefinition } from '@gooddata/sdk-model';
 import { IInsightWidget } from '@gooddata/sdk-backend-spi';
 import { IInsightWidgetDefinition } from '@gooddata/sdk-backend-spi';
+import { IKpi } from '@gooddata/sdk-model';
```

Comment:
I think the Legacy part was intentional, in the long run I think we still want to remove the KPI and replace it with a Headline (once there are alerts on insights mainly). So I'm not sure if we should remove the Legacy part here.

Reply:
I know. But it's there for 2 years we still have no plan for the removal. I think Legacy prefix does not make a good impression on customers in the end :) I can add comment to the interface, that it can be removed in the future, what do you think?

Reply:
Fair enough :)